compiler/sem/pragmas.nim

```diff
@@ -129,6 +129,10 @@ const
   allRoutinePragmas* = methodPragmas + iteratorPragmas + lambdaPragmas
   enumFieldPragmas* = {wDeprecated}
 
+  allowsCustomPragma = {skVar, skLet, skParam, skField, skProc, skFunc,
```

Comment:
shouldn't this include iterator?

Reply:
I don't see a reason why they shouldn't. The set was just copied from the previous the code, without me adjusting it.

Reply:
Custom pragmas are now also allowed on iterators.